controls/SV-257805.rb

```diff
@@ -32,4 +32,17 @@
     it { should be_disabled }
     it { should be_blacklisted }
   end
+
+  config_files = command('find /etc/modprobe.conf /etc/modprobe.d/* -print0').stdout.split("\0")
```

Comment:
Why is this one a slightly different pattern compared to the rest?

Reply:
I'm not sure why I wrote it that way. It should be fixed now.